tests/testthat/fixtures/canham/make_canham.R

```diff
@@ -0,0 +1,77 @@
+#Define specific functions for chosen DE, distribution of parameters, error process
+#Required library for multivariate normal
+require(mnormt)
```

Comment:
Can you add `mnormt` in the `DESCRIPTION` file under 'Suggests' and remove this `require` call please?

Reply:
Any functions from the `mnormt` package will need to be specified as `mnormt::funct()`

Reply:
Done.